back-end/helpers/transcript_data_formatter.js

```diff
@@ -1,7 +1,8 @@
 
 export default class TranscriptFormatter{
     
-    static async formatTranscipt(rawTranscript){
+    static async formatTranscipt(jsonTranscript){
+        const rawTranscript = JSON.parse(jsonTranscript);
```

Comment:
I suspect this will not fix the issue (since it is one of the things I tried). Unless you tested it manually and got a different result. I previously parsed the json and passed it to formatTranscript. The issue is more complex and is likely external to this component. We can talk about it more later today if you want.

Reply:
I actually think the initial parse should be external of this component (and parse should always be wrapped in a try-catch), but I am open to discussion.

Reply:
Ok yes sure. We can talk about it later today.  I didn't use the front end. I just passed the json file to it and it worked.